client/hwlib/debian/tests/run-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
As standard pratice use:
```
set -xeu
```

This will let you know when something fails or when a typo in a variable causes an empty expansion somewhere.

Reply:
added